keyboards/lily58/keymaps/paulomp90/keymap.c

```diff
@@ -0,0 +1,237 @@
+    *                   `-------------------''-------'           '------''--------------------'
+    */
+
+    [_RAISE] = LAYOUT(
+        _______, KC_F2,   KC_F3,   KC_F4,   KC_F5,   KC_F6,                     KC_F7,   KC_F8,   KC_F9,   KC_F10,  KC_F11, _______,
+        KC_F1,   _______, _______, _______, _______, _______,                   _______, XXXXXXX, KC_UP,  XXXXXXX, _______,  KC_F12,
+        _______, KC_END, KC_HOME, KC_PGDN, KC_PGUP, _______,                   XXXXXXX, KC_LEFT, KC_DOWN, KC_RGHT, XXXXXXX, _______,
+        _______, _______, _______, _______, _______, _______,   _______, _______, _______, _______,  _______, _______, _______, _______,
+                                    _______, _______, _______,  _______, _______,  _______, _______, _______
+    ),
+
+    /* ADJUST
+    * ,-----------------------------------------.                    ,-----------------------------------------.
+    * |      |      |      |      |      |      |                    |      |      |      |      |      |      |
+    * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+    * |      |      |      |      |      |      |                    |      |      |      |      |      |      |
+    * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+    * |RGB ON| HUE+ | SAT+ | VAL+ |      |      |-------.    ,-------|      |      |      |      |      |      |
+    * |------+------+------+------+------+------|       |    |       |------+------+------+------+------+------|
+    * | MODE | HUE- | SAT- | VAL- |      |      |-------|    |-------|      |      |      |      |      |      |
+    * `-----------------------------------------/       /     \      \-----------------------------------------'
+    *                   | LAlt | LGUI |LOWER | /Space  /       \Enter \  |RAISE | RGUI | RAlt | 
+    *                   |      |      |      |/       /         \      \ |      |      |      |
+    *                   `----------------------------'           '------''--------------------'
+    */
+
+    [_ADJUST] = LAYOUT(
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        RGB_TOG, RGB_HUI, RGB_SAI, RGB_VAI, XXXXXXX, XXXXXXX,                   XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+        RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX, XXXXXXX,
+                                    _______, _______, _______, _______, _______,  _______, _______, _______
+    )
+};
+
+layer_state_t layer_state_set_user(layer_state_t state) {
+    return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
```

Comment:
The core [tri layer](https://docs.qmk.fm/#/feature_tri_layer) feature should be used here instead.
```suggestion
```

Reply:
Applied, thanks for your suggestion